frontend/src/scenes/organization/Settings/InviteModal.tsx

```diff
@@ -65,102 +71,257 @@ function InviteRow({ index, isDeletable }: { index: number; isDeletable: boolean
+                        Invite team members
+                        <LemonButton
+                            icon={<IconClose />}
+                            type="stealth"
+                            size="small"
+                            disabled={loading}
+                            onClick={() => {
+                                resetInviteRows()
+                                onClose()
+                            }}
+                        />
+                    </div>
+                }
+            >
+                <h1 className="fw-800">Invite others to PostHog</h1>
+                {onboardingSidebarEnabled && !preflight?.email_service_available && (
                     <p>
-                        An invite is <b>specific to an email address</b> and <b>expires after 3 days</b>.
-                        <br />
-                        Name can be provided for the team member's convenience.
+                        This PostHog instance isn't configured to send emails. In the meantime, you can generate a link
+                        for each team member you want to invite. You can always invite others at a later time.
                     </p>
+                )}
+                <p>
+                    Invites <b>expire after 3 days</b>.
+                </p>
+                <LemonDivider dashed thick />
+                {preflight?.licensed_users_available === 0 && (
+                    <Alert
+                        type="warning"
+                        showIcon
+                        message={
+                            <>
+                                You've hit the limit of team members you can invite to your PostHog instance given your
+                                license. Please contact <a href="mailto:sales@posthog.com">sales@posthog.com</a> to
+                                upgrade your license.
+                            </>
+                        }
+                    />
+                )}
+                <div className="bulk-invite-modal">
                     <Row gutter={16}>
-                        <Col xs={areInvitesDeletable ? 11 : 12}>
+                        <Col xs={areInvitesDeletable || onboardingSidebarEnabled ? 11 : 12}>
                             <b>Email address</b>
                         </Col>
-                        <Col xs={areInvitesDeletable ? 11 : 12}>
+                        <Col xs={areInvitesDeletable || onboardingSidebarEnabled ? 11 : 12}>
                             <b>
-                                Name <i>(optional)</i>
+                                {onboardingSidebarEnabled && !preflight?.email_service_available
+                                    ? 'Share link'
+                                    : 'Name (optional)'}
                             </b>
                         </Col>
                     </Row>
 
+                    {onboardingSidebarEnabled &&
+                        invites.map((invite: OrganizationInviteType) => {
```

Comment:
These are the pending invites shown during ingestion so the user doesn't have to break out of the flow to view who's been invited. You cannot edit pending invites and can only delete/cancel them.